doc/sphinx/addendum/omega.rst

```diff
@@ -31,6 +31,8 @@ Description of ``omega``
 
 .. tacn:: omega
 
+   .. deprecated:: 8.12
```

Comment:
```suggestion
   .. deprecated:: 8.12

      Use :tacn:`lia` instead.
```

You should also adjust the warning above, which currently says that `omega` is "about to be deprecated".

Reply:
Done. Thanks.